docs/alerts/webhook-connections/aws-lambda.md

```diff
@@ -13,54 +13,51 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
 
 For example, you can create a Monitor that triggers a Lambda function when too many requests are received from a suspicious IP address. The Lambda function can shut down additional requests from that IP address, while simultaneously sending a notification to the security team for review.
 
-## Use Lambda Function URL or build an API in the API Gateway to expose a Lambda function
+## How to expose a Lambda function
+
+To expose a expose a Lambda function, you can either call a Lambda Function URL or build an API in the API Gateway.
```

Comment:
Double "expose a" here. One to be removed?
"call a Lambda Function URL" needs to be replaced; I would rather say "enable Lambda Function URL"
<img width="1957" alt="image" src="https://user-images.githubusercontent.com/108746237/230338038-18e6002f-b876-4908-b365-58283354eef4.png">